js/main.js

```diff
@@ -28,3 +30,92 @@ window.addEventListener('beforeunload', function () {
   var dataProfileJson = JSON.stringify(data.profile);
   localStorage.setItem('profile', dataProfileJson);
 });
+
+document.addEventListener('DOMContentLoaded', function () {
+  var userData = localStorage.getItem('profile');
+  if (userData !== null) {
+    data.profile = JSON.parse(userData);
+  } if (data.profile.username === '') {
```

Comment:
Move this condition down to a new line. With the formatting like this, it looks like an `else if`